File: app/src/main/java/org/pcsoft/app/jmc/exceptions/JmcDecoderException.java
package org.pcsoft.app.jmc.exceptions;

public class JmcDecoderException extends JmcException {
    public JmcDecoderException() {
    }

    public JmcDecoderException(String message) {
        super(message);
    }

    public JmcDecoderException(String message, Throwable cause) {
        super(message, cause);
    }

    public JmcDecoderException(Throwable cause) {
        super(cause);
    }
}

File: commons/src/main/kotlin/org/pcsoft/app/jmc/commons/exceptions/JmcDecoderException.kt
package org.pcsoft.app.jmc.commons.exceptions

class JmcDecoderException : JmcException {
    constructor()
    constructor(message: String?) : super(message)
    constructor(message: String?, cause: Throwable?) : super(message, cause)
    constructor(cause: Throwable?) : super(cause)
}